CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+### v5.0.0-rc.2 (2024-04-22)
```

Comment:
Update this to release date.

Reply:
I think it's fine to do a PR for this, but what I always do is just add it manually right before doing a release and then push to main 😎 

That way you don't have to guess the date ahead of time.

Reply:
Yeah but then you couldn't critique my English :-)